uast/uast.go

```diff
@@ -33,6 +33,12 @@ const (
 	// File is the root node of a single file AST.
 	File
 
+	// BooleanOperators
```

Comment:
Section names in comments, if any, should not go together with a constant. Otherwise, it shows up as the documentation of that constant.